### PR TITLE
Update antarian-ranger.md

### PR DESCRIPTION
**Pull request by jetfire158**
assuming we will use Legends since canon info is kind of non-existent.

### CODE REVIEW

**Review by jetfire158 — COMMENTED** [review]
curse the spacing!!

**Review by jetfire158 — COMMENTED** [review]
wordsmithing

File: content/theme/characters/archetypes/republic/antarian-ranger.md [hunk 1]
```diff
@@ -2,5 +2,42 @@
 title: "Antarian Ranger"
 date: 2020-11-14T18:42:45-08:00
 draft: false
+layout: archetype 
+
 ---
```
Comment (jeffhuyghebaert) [inline content/theme/characters/archetypes/republic/antarian-ranger.md]: remove line, fix spacing below

**Review by jetfire158 — COMMENTED** [review]
fixed

**Review by jeffhuyghebaert — APPROVED** [review]

**Pull request merged**